### PR TITLE
Revise API Documentation for Improved Clarity and Consistency

### PR DESCRIPTION
**Pull request by NewtTheWolf**
This update enhances the API documentation with several key improvements for clarity and usability. Please refer to the commit history for detailed changes.

#### Summary of updates:
- **Invalid Data Corrections:** Fixed previously invalid data references.
- **Example Cleanup:** Removed an unnecessary example to maintain consistency across sections.
- **Endpoint Additions:** Documented the new log endpoint.
- **Duplicated Entries:** Removed duplicated entries for clarity.

### CODE REVIEW

**Comment by NewtTheWolf** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/guides/prompt-manager/api-access.mdx [hunk 1]
```diff
@@ -156,7 +156,10 @@ The previously described `POST /projects/{projectId}/versions/{versionUuid}/docu
   "messages": [
     {
       "role": "user" | "system" | "assistant",
-      "content": string
+        {
+          "type": "text", //atm only text is supported
```
Comment (geclos) [inline docs/guides/prompt-manager/api-access.mdx:160]: this is not true? you can pass tool results for example

File: docs/guides/prompt-manager/api-access.mdx [hunk 2]
```diff
@@ -237,17 +199,9 @@ Retrieve a specific prompt by its path.
 **Path Parameters:**
 
 - `projectId`: Your project ID (required)
-- `versionUuid`: Version UUID (optional, defaults to 'live')
+- `versionUuid`: Version UUID (required, optional for SDK's defaults to 'live')
```
Comment (geclos) [inline docs/guides/prompt-manager/api-access.mdx:202]: i don't get this string change, it's not required 😅

File: docs/guides/prompt-manager/api-access.mdx [hunk 3]
```diff
@@ -257,58 +211,94 @@ The response contains the document details along with its configuration.
...
-  "evaluationUuids": ["evaluation-uuid-1", "evaluation-uuid-2"] // optional, defaults to all evaluations connected to the conversation prompt
+  "path": "path/to/document",
+  "messages": [
+    {
+      "role": "user" | "system" | "assistant",
+        {
+          "type": "text", //atm only text is supported
+          "content": string
+        }
+    }
```
Comment (geclos) [inline docs/guides/prompt-manager/api-access.mdx:281]: this is not true you can pass any message accepted by [vercel's ai sdk](https://sdk.vercel.ai/docs/reference/ai-sdk-core/stream-text#api-signature) (which follows OAI's format)

File: docs/guides/prompt-manager/api-access.mdx (hunk 2 above)
Comment (NewtTheWolf) [reply docs/guides/prompt-manager/api-access.mdx:202]: in the API the Parameter is required, you cannot just put nothing in it, you MUST pass at least "live" in it

thats why i did the change, but we also can revert it ^^

Comment (geclos) [reply docs/guides/prompt-manager/api-access.mdx:202]: ooooh i see, yeah in the api it's required you are totally right, it's the  sdk that does not require it 👌🏼

File: docs/guides/prompt-manager/api-access.mdx (hunk 3 above)
Comment (NewtTheWolf) [reply docs/guides/prompt-manager/api-access.mdx:285]: fixxed with new commit

**Comment by geclos** [comment]
thanks!

**Pull request merged**